web/src/screens/JobIndex.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import {Container, Col, Row,} from 'react-bootstrap'
+// import {jobShowHeader as Header, jobShowCarousel as JobGraphics, jobShowSummary as Summary } from '../components/Cards'
```

Comment:
You don't need to keep imports that aren't being used. Adding import statements can be done automatically.